rostsd_gen/index.js

```diff
@@ -270,7 +324,8 @@ function saveMsgFieldsAsTSD(
   indent = 0,
   lineEnd = ',',
   typePrefix = '',
-  useSamePackageSubFolder = false
+  useSamePackageSubFolder = false,
+  descriptorInterfaceType = false
```

Comment:
I prefer renaming `descriptorInterfaceType` to `willGenerateDescriptionInterface`